<?xml version="1.0" encoding="UTF-8"?>
<!-- SPDX-FileCopyrightText: Copyright 2024 shadPS4 Emulator Project
     SPDX-License-Identifier: GPL-2.0-or-later -->
<ui version="4.0">
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About shadPS4</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/shadps4.ico</normaloff>:/images/shadps4.ico</iconset>
  </property>
  <widget class="QLabel" name="shad_logo">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>271</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/images/shadps4.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="shad_title">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>15</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string notr="true">shadPS4</string>
   </property>
  </widget>
  <widget class="QLabel" name="shad_text">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>451</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>shadPS4 is an experimental open-source emulator for the PlayStation 4.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="shad_text_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>451</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>This software should not be used to play games you have not legally obtained.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="image_1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/images/github.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="image_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/images/discord.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="image_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>210</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/images/youtube.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="image_4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>210</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/images/ko-fi.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
   <widget class="QLabel" name="image_5">
     <property name="geometry">
       <rect>
         <x>670</x>
         <y>210</y>
         <width>80</width>
         <height>80</height>
       </rect>
     </property>
     <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
       <string/>
     </property>
     <property name="pixmap">
       <pixmap>:/images/website.png</pixmap>
     </property>
     <property name="scaledContents">
       <bool>true</bool>
     </property>
   </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
